### ISSUE TITLE
 Team 1 - Korrektur des ER-Modells/Datenbankschemas ist erfolgt

### ISSUE THREAD

**Issue by a71na** [IU-Internationale-Hochschule-Augsburg/fallstudie-daten-analyse-thema-ii#26]
Hi Ihr zwei :-).

ich habe gestern an der Datenbank etwas weiter programmiert. Dabei ist mir aufgefallen, dass der Begriff "User" scheinbar reserviert ist. Deswegen heißt unsere Tabelle jetzt: SURVEY_USER.

Nur damit ihr Bescheid wisst :-).

Liebe Grüße
Team 1

Das aktuellste Modell:
![Entity-Relationship-Modell-Onlineumfrage Korrektur](https://github.com/IU-Internationale-Hochschule-Augsburg/fallstudie-daten-analyse-thema-ii/assets/166111939/2cc6610d-eed1-41b8-9894-33c9aef8deaf)

P.S.: Ich glaube, ich habe keine Berechtigung bei euch selbst Labels zu setzen. :-)

**Comment by a71na** [comment]
Unter Settings -> Collaborators and teams kann man Personen hinzufügen und Rechte geben soweit ich weiß

**Comment by a71na** [comment]
Aus der Tabelle Question haben wir heute (28.04.2024) noch das Attribut Textinput herausgelöscht. Das benötigen wir nur für die Antworten in der Answer-Tabelle.

**Comment by a71na** [comment]
Heute habe ich nochmal eine Korrektur in der Datenbank vorgenommen. Wir können jetzt endlich einzelne Fragen hinzufügen und bearbeiten. Siehe Update hier: https://github.com/IU-Internationale-Hochschule-Augsburg/fallstudie-thema-2-online-umfragesystem/wiki/Datenbankstruktur-(inkl.-ER%E2%80%90Modell)

Letztendlich hat die Tabelle "QUESTION" das Attribut "DESCRIPTION" dazubekommen und die Antwortoptionen haben sich geändert. Es gibt nicht mehr "RADIOBUTTON1-10" und "CHECKBOX1-10", heißt wir unterscheiden zwischen den Antwortotpionen nur noch über den Fragetyp. Somit gibt es nur noch "ANSWER_OPTION1-10". Eine Modellierung zur Verdeutlichung folgt.

(Darüber hinaus haben wir die Wiki-Seite für die Grenzen und Einschränkungen upgedatet: https://github.com/IU-Internationale-Hochschule-Augsburg/fallstudie-thema-2-online-umfragesystem/wiki/Grenzen-und-Einschr%C3%A4nkungen)

**Comment by a71na** [comment]
![2024-05-19 Entity-Relationship-Modell-Onlineumfrage drawio](https://github.com/IU-Internationale-Hochschule-Augsburg/fallstudie-daten-analyse-thema-ii/assets/166111939/b45b99e0-df4e-4416-bf75-75820f698848)

**Comment by a71na** [comment]
Ich denke, dass wir die Tabelle "ANSWER" ebenfalls etwas anpassen müssen. Updates folgen, sobald ich das gemacht habe.

**Comment by a71na** [comment]
Die Tabelle Answer wurde ebenfalls angepasst :-)
![2024-05-23 Entity-Relationship-Modell-Onlineumfrage drawio](https://github.com/IU-Internationale-Hochschule-Augsburg/fallstudie-daten-analyse-thema-ii/assets/166111939/9a1eaeaf-0e11-4d6e-98a0-df21f54a5a9c)

**Comment by 12Jacky** [comment]
Hey eine Kurze Frage, das heißt es kommen nur einzelne Buchstaben rein und keine Zahlen?
Kannst du mir das Morgen dann vielleicht nochmal Zeigen, nicht das ich es jetzt falsch verstehe. Danke☺️

**Comment by a71na** [comment]
Richtig, das erkläre ich nochmal. Es sollen aber vmtl nur Y und N rein. Dafür, ob das entsprechende Kästchen angekreuzt ist oder nicht. Ist bisher eine Idee. Umsetzung folgt 😊

**Comment by 12Jacky** [comment]
Super danke. Gibt es nur Single choice? Muss mir dann etwas einfallen lassen, sonst sind in meinen Daten nur noch Y und N und nicht mehr schwimmen und Radfahren

**Comment by a71na** [comment]
Ah ja, da hast du recht. Meine Idee war es, die Daten aus der Tabelle Question mit der questionId zu laden. Wenn das aber zu umständlich ist, dann löse ich das gerne anders 🥰

**Comment by 12Jacky** [comment]
Es wäre super, wenn du magst und es Zeitlich schaffst mir nochmal die datenbank zu zeigen mit den Neuen Antwortmöglichkeiten. Ich würde dann das Säulendiagramm mal testen mit den Daten aus der Datenbank, wenn wir jetzt nur y  und n haben. Denke nur falls da Probleme auftauchen wäre es nicht schlecht wenn wir das zusammen mal Probieren. Gerne auch mit den Jungs wenn es sie Interessiert. 

Und falls du Langeweile hast würde ich ganz gerne schauen wie es ist mit Zahlen. 
Und vielleicht fällt dir ein wie wir das dann sinnvoll mit der Heatmap machen, da ich da gedanklich hänge wenn wir nur y und n eintrage dann hätte ich ja nur die zwei aber einen Richtigen Trend hätte ich dann wenn ich zwei Sachen verknüpfe, z.b. Alter und Raucher.  und die kombinierten Questions bekomme ich von euch aus der Datenbank(Wollte jetzt nur sichergehen das wir da auf dem selben stand sind. und wie genau das dann abgespeichert wird)

**Comment by a71na** [comment]
Klar, da können wir gerne noch einmal sprechen :)
